week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -18,24 +18,36 @@
 
 let array = [12, 73, 92, 45, 100, 14, 61];
 
-array.some((value) => {return (value % 2 == 0)}); /* this will return true as SOME values 
+array.some((value) => {
+  return value % 2 == 0;
+}); /* this will return true as SOME values 
 will have a remainder of 0 i.e. they are even numbers*/
 
-array.every((value) => {return (value % 2 == 0)}); /* this will return false as not ALL 
+array.every((value) => {
+  return value % 2 == 0;
+}); /* this will return false as not ALL 
 values will have a remainder of 0 i.e. there are some odd numbers in the array too*/
 
 /* + + + + + + + + + + + + + + */
 
 /* Now try to complete the exercise */
 
-function bushChecker() {
-
+function bushChecker(par) {
+  if (
+    par.some((H) => {
+      return H === "neon";
+    })
+  ) {
+    return "Toxic! Leave bush alone!";
+  } else {
+    return "Bush is safe to eat from";
+  }
 }
```

Comment:
You have used the wrong method and what you are looking for is Berries which are pink.